01-Fundamentals-part-2/assignments2.js

```diff
@@ -21,4 +21,22 @@ console.log("\n");
 console.log("Testing Data 2:");
 avgDolphins = calcAverage(85, 54, 71);
 avgKoalas = calcAverage(23, 34, 27);
-checkWinner(avgDolphins, avgKoalas);
+checkWinner(avgDolphins, avgKoalas);*/
+
+function calcTip(billValue) {
+  if (50 < billValue < 300) {
+    //console.log(`The tip is ${billValue * 0.15}$`);
+    return billValue * 0.15;
+  } else if (billValue <= 50 && billValue >= 300) {
```

Comment:
in this case you have two branches inside if: 1) between 50 and 300 and 2) other value. Thus, in the second branch, you should just use `else` without specifying additional conditions. In general, if your function is supposed to return something (not void), make sure to cover all the conditions (e.g. using `else` at the end) to avoid bugs that would be hard to discover